### PR TITLE
Corrected data file format and added data analysis code

### PR DESCRIPTION
**Pull request by JiaJunHuang120000**

### CODE REVIEW

**Comment by JiaJunHuang120000** [comment]
Added the analysis codes and changed the names of some data fomatting for the code to access.

File: Sipm_RadTest/Davis/FlowerBoard/IV_PreIrradiation/Board62_1_IV_UTC_20240514__17_47.txt
Comment (bschmookler) [inline Sipm_RadTest/Davis/FlowerBoard/IV_PreIrradiation/Board62_1_IV_UTC_20240514__17_47.txt:1]: Hi @JiaJunHuang120000. Why did you delete these pre-irradition FlowerBoard files and are not replacing them wit other ones?

File: Sipm_RadTest/FlowerBoard/Irradiated/Board62_1_IV_UTC_20240514__23_45.txt
Comment (bschmookler) [inline Sipm_RadTest/FlowerBoard/Irradiated/Board62_1_IV_UTC_20240514__23_45.txt:1]: Why do you want to move this out of the 'Davis' subfolder?

File: Sipm_RadTest/Davis/S13_6050/IV_PreIrradiation/S13A10_IV_UTC_20240509__22_28.txt
Comment (bschmookler) [inline Sipm_RadTest/Davis/S13_6050/IV_PreIrradiation/S13A10_IV_UTC_20240509__22_28.txt:1]: Again, you delete this file but don't replace it with anything???

File: Sipm_RadTest/JF_6015/Irradiated/6015_1_IV_UTC_20240516__01_37.txt
Comment (bschmookler) [inline Sipm_RadTest/JF_6015/Irradiated/6015_1_IV_UTC_20240516__01_37.txt:1]: Same comment as above.

**Review by bschmookler — CHANGES_REQUESTED** [review]
Check comments.

**Pull request closed without merging**